src/main/java/com/mastery/java/task/controllers/EmployeeRestController.java

```diff
@@ -0,0 +1,114 @@
+package com.mastery.java.task.controllers;
+
+import com.mastery.java.task.dto.Employee;
+import com.mastery.java.task.exceptions.DuplicateEmployeeException;
+import com.mastery.java.task.exceptions.InvalidDateException;
+import com.mastery.java.task.service.EmployeeService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jms.core.JmsMessagingTemplate;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.jms.Queue;
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@Api("Employee rest")
+public class EmployeeRestController {
+
+    private static final Logger log = Logger.getLogger(EmployeeRestController.class);
+    private static final Log logCon = LogFactory.getLog(EmployeeRestController.class);
+    private final EmployeeService employeeService;
+    private final JmsMessagingTemplate jmsMessagingTemplate;
+    private final Queue queue;
+
+    @Autowired
+    public EmployeeRestController(EmployeeService employeeService, JmsMessagingTemplate jmsMessagingTemplate, Queue queue) {
```

Comment:
please consider the possibility of using Lombok

Reply:
add Conctructoe annotation from controller using Lombok